schemas/user.py

```diff
@@ -45,26 +45,23 @@ def validate_email(cls, v):
         message = False
 
         if not v:
-            message = "Електронний адрес не може бути порожнім"
+            message = "The email address cannot be empty."
 
         elif not re.fullmatch(regex, v):
-            message = f"Невірний формат адресу електронної пошти: {v}."
+            message = f"Invalid email address format: {v}."
 
         if message:
-            raise ValueError(message)
-
+            raise BackendException(message=message)
         return v
 
     @validator('password_re_check')
-    def validate_password(cls, v, values):
+    def validate_password(cls, v, values):  # TODO may be there is need to use classmethod decorator
         password = values.get('password')
 
         if not password or not v:
-            raise ValueError('Паролі не можуть бути порожніми')
-
+            raise BackendException(message='Passwords cannot be empty.')
```

Comment:
Don't use `BackendException` inside Pydantic schemas validation. it should be handled by another handler: `validation_exception_handler`.